webrtc/guides/callFlow.md

```diff
@@ -0,0 +1,31 @@
+## Call Flow
+
+In this diagram we will review the interactions you will need in order to establish a working WebRTC session. To simplify the diagram, we are showing only one web browser in the session, but adding more merely follows the same pattern.
+
+
+![alt_text](../../images/webrtc_callflow1.png "image_tooltip")
+
+
+We start by creating a Session (1) and a Participant (2), and then adding the Participant to the Session (3) - when we do so, we will be specifying a Subscription to the SessionId within the _body_ parameter of the addParticipantToSession() call.
+
+Next we serve a page to the user that has the required WebRTC components as well as the JWT that we received when we [made what call???].
+
+Within our browser javascript implementation, we will be calling connect() (5) in order to establish our connection to the WebRTC platform. Once that is complete, we can call publish() (6) to start flowing media. 
```

Comment:
Perhaps change connect() to BandwidthRtc.connect(), and possibly link it to https://www.npmjs.com/package/@bandwidth/webrtc-browser#connect